pennylane_forest/wavefunction.py

```diff
@@ -152,25 +152,44 @@ def ev(self, A, wires):
           float: expectation value :math:`\left\langle{A}\right\rangle = \left\langle{\psi}\mid A\mid{\psi}\right\rangle`
         """
         # Expand the Hermitian observable over the entire subsystem
-        A = self.expand_one(A, wires)
+        A = self.expand(A, wires)
         return np.vdot(self.state, A @ self.state).real
 
-    def expand_one(self, U, wires):
-        r"""Expand a one-qubit operator into a full system operator.
+    def expand(self, U, wires):
+        r"""Expand a multi-qubit operator into a full system operator.
```

Comment:
Most (if not all) of the comments I made for `numpy_wavefunction.py` apply here too.

Reply:
Copied over the fixed function :slightly_smiling_face: